wp-config-sample.php

```diff
@@ -57,13 +48,195 @@
+// define( 'FS_METHOD', 'ftpext' ); // "direct", "ssh2", "ftpext", or "ftpsockets".
+// define( 'FTP_BASE', __DIR__ . CP_CORE_PATH );
+// define( 'FTP_CONTENT_DIR', __DIR__ . CP_CONTENT_PATH );
+// define( 'FTP_PLUGIN_DIR ', WP_PLUGIN_DIR );
+// define( 'FTP_PUBKEY', '/home/username/.ssh/id_rsa.pub' );
+// define( 'FTP_PRIKEY', '/home/username/.ssh/id_rsa' );
+// define( 'FTP_USER', 'username' );
+// define( 'FTP_PASS', 'password' );
+// define( 'FTP_HOST', 'ftp.example.org' );
+// define( 'FTP_SSL', true );
+//
+// ** Control the psuedo-cron built into ClassicPress **
+// ** This allows disabling so an external cron service like
+// ** SetCronJob.com can be used,
+// define( 'DISABLE_WP_CRON', true );
+// ** This allows cron to run on a redirect rather than at
+// ** the end of a page load. Only use as last resort.
+// define( 'ALTERNATE_WP_CRON', true );
+// ** Ensure cron can only one once ever "n" seconds.
+// define( 'WP_CRON_LOCK_TIMEOUT', 300 );
+//
+// ** Delete trashed posts ever "n" days **
+// define( 'EMPTY_TRASH_DAYS', 30 );
+//
+// ** Stop end-users from adding plugins or changing themes **
+// define( 'DISALLOW_FILE_MODS', true );
+//
+// ** Stop editing of plugins or themes from within the admin **
+// define( 'DISALLOW_FILE_EDIT', true );
+//
+// ** Secure the wp-admin and login via HTTPS, respectively **
+// define( 'FORCE_SSL_ADMIN', true );
+// define( 'FORCE_SSL_LOGIN', true );
+//
+// ** Block outgoing HTTP requests, except for those whitelisted **
+// define( 'WP_HTTP_BLOCK_EXTERNAL', true );
+// define( 'WP_ACCESSIBLE_HOSTS', 'api.classicpress.org,*.github.com' );
+//
+// ** Disable automatic updater from updating core, plugins, themes, etc.
+// define( 'AUTOMATIC_UPDATER_DISABLED', true );
+//
+// ** Disable automatic update of specific types of core updates **
+// ** Disable all core updates
+// define( 'WP_AUTO_UPDATE_CORE', false );
+//
+// ** Enable all core updates, including minor and major
+// define( 'WP_AUTO_UPDATE_CORE', true );
+//
+// ** Enable core updates for minor releases (default):
+// define( 'WP_AUTO_UPDATE_CORE', 'minor' );
+//
+// ** Overwrite image edits rather than keep all edits
+// define( 'IMAGE_EDIT_OVERWRITE', true );
+//
+// ** No unfiltered HTML, even Administrator and Editor roles
+// define( 'DISALLOW_UNFILTERED_HTML', true );
+//
+// ** Set the theme ClassicPress will default to
+// define('WP_DEFAULT_THEME', 'my-custom-theme');;
+//
+// NOTE: The above are not all potential options. Plugins and
+// themes often have their own config settings, and there are
+// a several more obscure settings buried in ClassicPress.
+// So check the docs and/or Google to discover more.
+//
+
+/*----------------------------------------------------*
+ *  END-CONFIG - Add your configuration changes ABOVE
+ *  What follows is ClassicPress default config.
+ * ---------------------------------------------------*/
+
+/** MySQL hostname */
+if ( ! defined( 'DB_HOST' ) ) {
+	define( 'DB_HOST', 'localhost' );

```

Comment:
Note for when we revisit the configuration system: There is some code in the installation process that deals with an existing `wp-config.php` file.  It would be broken by these changes, first because it doesn't understand the conditionals, and second because of the spacing:

https://github.com/ClassicPress/ClassicPress/blob/4ba650b296ce881a8d764dd3a5bcada8d1cd5fb5/src/wp-admin/setup-config.php#L327-L363